main.go

```diff
@@ -255,6 +255,8 @@ func run(ctx context.Context, wg *sync.WaitGroup, globalCollectionOpts state.Col
 		wg.Done()
 	}, logger, "high frequency query statistics of all servers", schedulerGroups["stats"])
 
+	runner.SetupWebsocketForAllServers(ctx, servers, globalCollectionOpts, logger)
```

Comment:
Should we have some sort of "close websocket" shutdown? This `run` func is returning the `shutdown`), we could tweak/reuse that.

Reply:
Shutdown should happen automatically AFAIK because the websocket runner calls `context.WithCancel`. I could be wrong though.

Reply:
Ah yeah just connected the dots, you created `connCtx, cancelConn := context.WithCancel(ctx)` and doing `server.WebSocket.Close()` within `<-connCtx.Done()` so it should be fine.
We could write `defer cancelConn()` and `defer conn.Close()` over there also but I guess it's somewhat optional given they are handled within `<-connCtx.Done()`.